luigi/s3.py

```diff
@@ -284,23 +283,82 @@ def get_as_string(self, s3_path):
     def copy(self, source_path, destination_path, **kwargs):
         """
         Copy an object from one S3 location to another.
-
         :param kwargs: Keyword arguments are passed to the boto function `copy_key`
         """
         (src_bucket, src_key) = self._path_to_bucket_and_key(source_path)
         (dst_bucket, dst_key) = self._path_to_bucket_and_key(destination_path)
 
         s3_bucket = self.s3.get_bucket(dst_bucket, validate=True)
+        source_bucket = self.s3.get_bucket(src_bucket, validate=True)
+
+        multipart_size = 67108864
```

Comment:
I can add this as a comment in the commit as per your example, or try and dig back through the repo to see why this magic constant exists in the `put_multipart` method in the first place?